### PR TITLE
Add outbreak tooltip and fetch uncertainty chart data

### PR DESCRIPTION
**Pull request by shreeyash07**
- Addresses #36 #35 #86 

## Changes

- Tooltip over the outbreak card
- fetch uncertainty chart data
- fetch indicator name status component

## This PR doesn't introduce any:

- [x] temporary files, auto-generated files or secret keys
- [x] build works
- [x] eslint issues
- [x] typescript issues
- [x] codegen errors
- [x] `console.log` meant for debugging
- [x] typos
- [x] unwanted comments
- [x] conflict markers

### CODE REVIEW

File: app/components/UncertaintyChart/index.tsx [hunk 1]
```diff
@@ -12,16 +12,24 @@ import {
...
 } from '@the-deep/deep-ui';
 import { _cs } from '@togglecorp/fujs';
-
-import { rangeData } from '#utils/dummyData';
 import styles from './styles.css';
 
+export interface UncertainData {
+    indicatorValue?: string | null;
+    date: string;
+    uncertainRange: (string | undefined)[];
```
Comment (AdityaKhatri) [inline app/components/UncertaintyChart/index.tsx]: filter isDefined before

File: app/views/Dashboard/Country/index.tsx [hunk 2]
```diff
@@ -134,12 +167,20 @@ function Country(props: Props) {
 
     const countryVariables = useMemo((): CountryQueryVariables => ({
         iso3: filterValues?.country ?? 'AFG',
+        gte: '2021-08-01',
```
Comment (AdityaKhatri) [inline app/views/Dashboard/Country/index.tsx:171]: why this?

File: app/views/Dashboard/Country/index.tsx [hunk 3]
```diff
@@ -134,12 +167,20 @@ function Country(props: Props) {
...
         contextIndicatorId: 'total_cases',
         disaggregationIso3: filterValues?.country ?? 'AFG',
         emergency: filterValues?.outbreak,
+        uncertaintyIso3: filterValues?.country ?? 'AFG',
+        subvariable: filterValues?.subvariable ?? '',
+        indicatorId: filterValues?.indicator,
+        category: 'Global',
```
Comment (AdityaKhatri) [inline app/views/Dashboard/Country/index.tsx:179]: write note why this is static

**Review by AdityaKhatri — CHANGES_REQUESTED** [review]

File: app/views/Dashboard/Country/index.tsx (hunk 2 above)
Comment (subinasr) [reply app/views/Dashboard/Country/index.tsx:171]: Let's use the current date and calculate.

**Review by subinasr — APPROVED** [review]

**Pull request merged**